homeassistant/components/switch/rpi_gpio.py

```diff
@@ -83,13 +96,13 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """ Turn the device on. """
-        if self._switch(True):
+        if self._switch(True if self._active_state == "HIGH" else False):
```

Comment:
I had "invert" as the original parameter, and change it at the last minute before PR...lol

Of course it makes sense! Already submitted changes.

Thanks!